master/internal/command/authz_basic_impl.go

```diff
@@ -92,6 +92,13 @@ func (a *NSCAuthZBasic) CanTerminateTensorboard(
 	return nil
 }
 
+// CanCreateGenericTask always returns nil.
```

Comment:
not sure how familiar you are with EE process: we need to make ready a separate PR for EE implementing this in RBAC, to avoid breaking EE build when this is merged.

Reply:
https://github.com/determined-ai/determined-ee/pull/1187